sidekick_core/test/dart_command_test.dart

```diff
@@ -0,0 +1,56 @@
+import 'package:sidekick_core/sidekick_core.dart';
+import 'package:test/test.dart';
+
+import 'fake_sdk.dart';
+import 'init_test.dart';
+
+void main() {
+  test('dart command works when dartSdkPath is set', () async {
+    await insideFakeSidekickProject((dir) async {
+      final runner = initializeSidekick(
+        name: 'dash',
+        dartSdkPath: fakeDartSdk().path,
+      );
+      runner.addCommand(DartCommand());
+      await runner.run(['dart']);
+    });
+  });
+
+  test('dart command fails when dartSdkPath is not set', () async {
+    await insideFakeSidekickProject((dir) async {
+      final runner = initializeSidekick(
+        name: 'dash',
+        // ignore: avoid_redundant_argument_values
+        dartSdkPath: null,
+      );
+      runner.addCommand(DartCommand());
+      try {
+        await runner.run(['dart']);
+        fail('did not throw');
+      } catch (e) {
+        expect(e, isA<DartSdkNotSetException>());
+      }
```

Comment:
You can do this in just a single line
```suggestion
      expect(() => runner.run(['dart']), throwsA(isA<DartSdkNotSetException>()));
```